app/views/application/_menu.html.haml

```diff
@@ -0,0 +1,3 @@
+%ul.menu
+  %li{class: ('active' if current_page?(controller: 'books'))}= link_to 'Books', books_path
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [91/80]